Inc/ST-LIB_HIGH/Protections/Protection.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "C++Utilities/CppUtils.hpp"
+
+#define PROTECTIONTYPE_LENGTH 5
+
+enum ProtectionType : uint64_t {
+    BELOW = std::numeric_limits<uint64_t>::max() - PROTECTIONTYPE_LENGTH + 1,
+    ABOVE,
+    OUT_OF_RANGE,
+    EQUALS,
+    NOT_EQUALS
+};
+
+
+class Protection {
+public:
+
+    double* src;
+    char* name = nullptr;
+    vector<ProtectionType> protections;
+    vector<double> boundaries;
+    uint8_t boundary_counter = 0;
+    ProtectionType jumped_potection = (ProtectionType)0;
```

Comment:
```suggestion
    ProtectionType jumped_protection = (ProtectionType)0;
```
Añade la r a la variable